docs/user_guide/assessment.rst

```diff
@@ -118,25 +118,32 @@ across groups and also the difference and ratio between the maximum and minimum:
     ratio in recall =  0.0
 
 A single instance of :class:`fairlearn.metrics.MetricFrame` can evaluate multiple
-metrics simultaneously:
+metrics simultaneously (note that :func:`fairlearn.metrics.count` can be used to
+show each group's size):
 
 .. doctest:: assessment_metrics
     :options:  +NORMALIZE_WHITESPACE
 
-    >>> multi_metric = MetricFrame({'precision':skm.precision_score, 'recall':skm.recall_score},
+    >>> from fairlearn.metrics import count
+    >>> multi_metric = MetricFrame({'precision':skm.precision_score,
+                                    'recall':skm.recall_score,
```

Comment:
I believe you need to add `...` at the start of continuation lines. Try to run the doc build with `doctest` rather than `html` as the arg to see the doctest output. It should tell you what's wrong. The doc build which builds the webpage (separate from doctest) also failed but that's something we're tackling in #815 and that will probably get merged before your PR, hopefully tomorrow.